packages/x-date-pickers/src/DatePicker/shared.ts

```diff
@@ -25,6 +25,8 @@ export interface BaseDatePickerProps<TInputDate, TDate>
   onViewChange?: (view: CalendarPickerView) => void;
   /**
    * First view to show.
+   * Must be a valid option from `views` list
+   * @default 'day'
```

Comment:
From what I understand, the default is the first item of `views`

Reply:
Nope, we have `day = 'something'` for each of the pickers and I just copied their initial values.
We would fallback to first item of `views` if the requested `openTo` does not exist in `views`. 😉